### DISCUSSION TITLE
Add support for MiBoxer (Mi-Light LS2) in TuyaMCU

### DISCUSSION THREAD

**Discussion by ajvdw** [arendst/Tasmota#10368] [Ideas]
**Have you looked for this feature in other issues and in the docs?**
yes

**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is.
TuyaMCU does not work with LS2. LS2 uses a W3 MCU (non ESP) but can easily be replaced with an ESP8266.

**Describe the solution you'd like**
I'd like a TuyaSend0 (DPID-Type=0) command that is equal to TuyaSend3 but can send hex codes unchanged. For example TuyaSend0 101, 008006010000008080C8 will switch the power on. It sends the following data to the device <55 AA 00 06 00 10 65 00 00 0C 41 00 00 80 06 01 00 00 00 80 80 C8 16> 

**Describe alternatives you've considered**
Asking full support for this device is a bit too much.... But if you considering:
Actually, the TuyaMCU should  have structure where it supports product specific sub protocols. Since exchanging the product ID is the first step in every communication using Tuya. Every product can have its own sub-protocol.
Tuya offers their cloud services to hardware vendors - so they don't have to run the infrastructure. Look for example at tuyav on github

**Additional context**
Mi-Light L2 https://www.bol.com/nl/p/mi-light-controller-ls2-5-in-1-8-zone/9200000093930728

For anyone interested: I captured / sniffed the data and do have a working prototype using an ESP8266.

**Comment by vic42** [comment]
Sending a random payload shouldn't be overly complicated to implement. The TuyaSendCmd function seems to do exactly that. I'm just wondering about the benefit beyond a one-off use case.
